### PR TITLE
NO-TICKET Update default gitops-email and gitops-user

### PR DESCRIPTION
**Pull request by veerendra2**
### Type of Change

- [x] Bugfix
- [ ] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description
Update default gitops-email and gitops-user

### Checklist

- [x] Add relevant labels (for example type of change or patch/minor/major)
- [x] Make sure not to introduce some mistakes
- [x] Update documentation
- [x] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [x] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Comment by veerendra2** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/template_gitops.yml [hunk 1]
```diff
@@ -49,11 +49,11 @@ on:
       gitops-user:
         required: false
         type: string
-        default: "Staffbot"
+        default: "staffbase-actions[bot]"
```
Comment (0x46616c6b) [inline .github/workflows/template_gitops.yml:52]:
```suggestion
        default: "staffbase-actions"
```

I guess the bot suffix is auto attached by github and must not included in the username.

File: .github/workflows/template_gitops.yml
Comment (flaxel) [inline .github/workflows/template_gitops.yml:1]: Would be nice to update the documentation too. 🙌

Comment (veerendra2) [reply .github/workflows/template_gitops.yml:1]: Sorry, where is this documentation? in https://backstage.staffbase.com? Can you please point the doc link?

Comment (axdotl) [reply .github/workflows/template_gitops.yml:1]: I guess flaxel is talking about the `README.md`

Comment (veerendra2) [reply .github/workflows/template_gitops.yml:1]: Thanks. Done ✅

**Review by axdotl — APPROVED** [review]

**Review by flaxel — APPROVED** [review]

**Pull request merged**